.github/workflows/static_analysis.yaml

```
@@ -0,0 +1,21 @@
+name: Terraform Validation
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - infra/**
+      - .terraform-version
+      - .pre-commit-config.yaml
+      - .trivyignore
+
+jobs:
+  tf_analysis:
+    # Don't run in Changeset PRs
+    if: ${{ github.actor != 'dx-pagopa-bot' }}
```

I think that the author of changeset PRs is the default one
```suggestion
    if: ${{ github.actor != 'github-actions[bot]' }}
```